### PR TITLE
Flaw twitter style

### PR DESCRIPTION
**Pull request by gtrufitt**
## What does this change?

Makes the styling of the twitter non-enhanced embeds (always shown on mobile) look a little nicer.
### Before

screen shot 2016-09-23 at 11 45 42
### After

screen shot 2016-09-23 at 11 44 58

Can you take a look @zeftilldeath pls.thx.

### CODE REVIEW

**Comment by paperboyo** [comment]
Look better than originals. Enhance embeds as a sub-setting of Enhanced view setting in the future User preferences?

File: common/app/views/support/HtmlCleaner.scala [hunk 1]
```diff
@@ -298,7 +298,7 @@ class TweetCleaner(content: Content, amp: Boolean) extends HtmlCleaner {
           if (el.children.size > 1) {
             val body = el.child(0).attr("class", "tweet-body")
             val date = el.child(1).attr("class", "tweet-date")
-            val user = el.ownText()
+            val user = el.ownText().replaceFirst("— ", "")
```
Comment (dominickendrick) [inline common/app/views/support/HtmlCleaner.scala]: I thought you didn't know scala ?

**Comment by dominickendrick** [comment]
👍 🐦

File: common/app/views/support/HtmlCleaner.scala (hunk 1 above)
Comment (gtrufitt) [inline common/app/views/support/HtmlCleaner.scala]: I asked @rjmk

**Comment by rjmk** [comment]
Can we have a screenshot update before merge? 🙏

**Comment by gtrufitt** [comment]
After talking to @zeftilldeath the conclusion was I should start again and make the non-enhanced embeds look more like the native Twitter ones, they now look like this:

image

I also did some tidying like splitting tweets out from liveblog css and splitting username and id, so another lookSee would be good @guardian/dotcom-platform 

Ta.

File: static/src/stylesheets/module/content/_live-blog.scss [hunk 2]
```diff
@@ -419,85 +419,6 @@ $block-padding-right: $gs-gutter;
     }
 }
```
Comment (NataliaLKB) [inline static/src/stylesheets/module/content/_live-blog.scss]: 👍  x 1000 for removing this css from `live-blog.scss`

**Comment by NataliaLKB** [comment]
👍

**Comment by paperboyo** [comment]
Arial :barf:

**Comment by gtrufitt** [comment]
@paperboyo 💋

**Comment by paperboyo** [comment]
Yeah, I know, imitating closely a style of something that has no style. Tough.

File: static/src/stylesheets/module/content/_tweets.scss [hunk 3]
```diff
@@ -0,0 +1,101 @@
+/* Tweets
+
+    Non-upgraded tweets are styles like the upgraded twitter embed
+    so we have font-families and colours in here that are not part
+    of the Guardian styles.
+   ========================================================================== */
+.tweet {
+    display: table-cell;
+    width: 10000px;
```
Comment (rjmk) [inline static/src/stylesheets/module/content/_tweets.scss]: Does this merit a comment? Or is it a CSS trick people should know (what is the trick?)?

Comment (gtrufitt) [inline static/src/stylesheets/module/content/_tweets.scss]: dunno.. didn't add it, but seems weird... I'll take a look

**Pull request merged**